src/main/kotlin/com/github/warningimhack3r/intellijshadcnplugin/backend/helpers/DependencyManager.kt

```diff
@@ -35,12 +35,22 @@ class DependencyManager(private val project: Project) {
         }.values.firstOrNull()
     }
 
+    private fun getInstallCommand(packageManager: String): String {
+        return when (packageManager) {
+            "npm" -> "i"
+            "pnpm" -> "add"
+            "yarn" -> "add"
+            "bun" -> "add"
+            else -> throw IllegalArgumentException("Unknown package manager: $packageManager")
```

Comment:
Now that we got package managers as strings multiple times, I think creating an enum for that would be better, like what I did for `InstallationType`.
That way, we'll avoid having an `else ->` case with an exception as it wouldn't be possible anymore (as both the param and the return type of `getPackageManager` would be that enum)